src/App.js

```diff
@@ -1,21 +1,27 @@
-import React, { Component } from 'react';
```

Comment:
- [ ] Leave the `import React from 'react'` because otherwise you will face an error 
![image](https://user-images.githubusercontent.com/62705947/218503746-8103a1b0-ac92-4ead-8976-799ca3920f67.png)